implementation/contracts/deposit/DepositFunding.sol

```diff
@@ -141,15 +141,15 @@ library DepositFunding {
         fundingTeardown(_d);
     }
 
-    /// @notice                 Anyone can provide a signature that was not requested to prove fraud during funding
-    /// @dev                    ECDSA is NOT SECURE unless you verify the digest
-    /// @param  _d              deposit storage pointer
-    /// @param  _v              Signature recovery value
-    /// @param  _r              Signature R value
-    /// @param  _s              Signature S value
-    /// @param _signedDigest    The digest signed by the signature vrs tuple
-    /// @param _preimage        The sha256 preimage of the digest
-    /// @return                 True if successful, otherwise revert
+    /// @notice                 Anyone can provide a signature that was not requested to prove fraud during funding.
+    /// @dev                    ECDSA is NOT SECURE unless you verify the digest.
```

Comment:
This isn't really actionable. What does that mean to me as a dev who is calling `provideFundingECDSAFraudProof`? Is that an issue for me as the dev caller, or is it an issue for the keep implementor?

Reply:
agree. I think we should remove this entirely. We verify on keep, but it's not pertinent to developers using tbtc. Might be worth adding it in-line on ECDSAKeep if at all.

Reply:
Agree, //cc @nkuba on whether it's necessary on the Keep side.

It *may* be worth noting here that we expect the Keep to verify it, but could see that going either way.